guide/popular-topics/reactions.md

```diff
@@ -89,9 +99,13 @@ For custom emojis, there are multiple ways of reacting. Like Unicode emojis, you
 
 This format is essentially the name of the emoji, followed by its ID. Copy & paste the ID into the `.react()` method as a string.
 
-```js {2-4}
-client.on('messageCreate', message => {
-	if (message.content === '!react-custom') {
+```js {4-8}
+client.on('interactionCreate', async interaction => {
+	if (!interaction.isCommand()) return;
+
+	if (interaction.commandName === 'react-custom') {
+		interaction.reply('You can react with custom emojis!');
+		const message = await interaction.fetchReply();
```

Comment:
```suggestion
		const message = await interaction.reply('You can react with custom emojis!', { fetchReply: true });
```
```suggestion
		await interaction.reply('You can react with custom emojis!');
		const message = await interaction.fetchReply();
```

Reply:
Using the `fetchReply` option is preferred